grails-app/services/org/pih/warehouse/core/ApiClientService.groovy

```diff
@@ -34,8 +34,8 @@ class ApiClientService {
         return execute(new HttpGet(url))
     }
 
-    JSONObject post(String url, Map requestData) {
-        return execute(new HttpPost(url), requestData)
+    JSONObject post(String url, Map payload, Map headers) {
```

Comment:
Looks like this is used in the `RecaptchaService`. I think you should either add a default option for headers here or add an empty map there.

Reply:
Good point. I thought I had left the previous method, but I guess not. I like using a default option instead of chaining overriding methods so I'll go with that approach.